src/core/ext/filters/client_channel/subchannel_stream_client.cc

```diff
@@ -94,8 +95,10 @@ void SubchannelStreamClient::Orphan() {
     MutexLock lock(&mu_);
     event_handler_.reset();
     call_state_.reset();
-    if (retry_timer_callback_pending_) {
-      grpc_timer_cancel(&retry_timer_);
+    if (retry_timer_handle_.has_value()) {
+      connected_subchannel_->args().GetObject<EventEngine>()->Cancel(
```

Comment:
I've seen differing opinions here, so it's your call, but I might prefer to see the engine pointer cached locally on SubchannelStreamClient construction to save the lookup cost.

Reply:
done.